src/settings_type.h

```diff
@@ -551,6 +551,7 @@ struct EconomySettings {
 	uint16_t town_noise_population[4];         ///< population to base decision on noise evaluation (@see town_council_tolerance)
 	bool   allow_town_level_crossings;       ///< towns are allowed to build level crossings
 	bool   infrastructure_maintenance;       ///< enable monthly maintenance fee for owner infrastructure
+	uint8_t timekeeping_units;               ///< time units to use for the game economy, either calendar or wallclock
```

Comment:
Please make this an enum (of type uint8_t), so we don't have `0` and `1`, but actual readable things.

Reply:
See fa1e27994d for an example.

Reply:
You resolved this without actually resolving it :) It still isn't of the enum type.